magda-gateway/src/createBaseProxy.ts

```diff
@@ -44,5 +47,28 @@ export default function createBaseProxy(): httpProxy {
         });
     });
 
+    proxy.on("proxyReq", function(proxyReq, req, res) {
+        proxyReq.setHeader("TenantId", "undefined");
```

Comment:
Might want to make this `X-Magda-Tenant-Id` or something like that. As I understand headers that aren't part of the standard are supposed to have `X` in front of them. I don't think it usually matters but certain servers get angry and spew warning messages into the logs.

Reply:
done